lib/registry/src/package.rs

```diff
@@ -1,8 +1,17 @@
 use crate::PartialWapmConfig;
+use regex::Regex;
 use std::path::PathBuf;
 use std::{fmt, str::FromStr};
 use url::Url;
 
+const REGEX_FULL: &str = r#"^([a-zA-Z0-9\-_]+)/([a-zA-Z0-9\-_]+)(@([a-zA-Z0-9\.\-_]+*))?$"#;
```

Comment:
REGEX_PACKAGE_WITH_VERSION is more accurate

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3432/commits/8e865a241168661521ed252b247e42711c340c20